### PR TITLE
Add OSX ARM64 build

### PR DESCRIPTION
**Pull request by corydolphin**
Following the guide here for autoconf builds. 

https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/#how-to-add-a-osx-arm64-build-to-a-feedstock

Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/build.sh
Comment (jakirkham) [inline recipe/build.sh:18]: Would keep this here

**Review by jakirkham — COMMENTED** [review]
Would suggest reading [this blogpost]( https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/#how-to-add-a-osx-arm64-build-to-a-feedstock )

File: recipe/build.sh
Comment (corydolphin) [reply recipe/build.sh:18]: @jakirkham `make check` invokes the test suite. This is failing on non-native environment (e.g. which seems like it should live in `test`?

Comment (corydolphin) [reply recipe/build.sh:18]: Looking at `libuv` they took the conditional approach suggested 

image


https://github.com/conda-forge/libuv-feedstock/blob/main/recipe/build.sh#LL40-L42C3

Comment (jakirkham) [reply recipe/build.sh:18]: Yeah it is difficult to separate `make check` specifically from the build environment. So maintainers typically just handle this in the build with conditional checks (as noted)

File: recipe/meta.yaml
Comment (jakirkham) [inline recipe/meta.yaml]: Maybe worth converting these to `# [unix]` as well for consistency

**Comment by corydolphin** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml
Comment (corydolphin) [reply recipe/meta.yaml]: Updated! Thanks for the suggestion.

**Comment by corydolphin** [comment]
Hey @jakirkham Is there any chance you'd be able to review/approve/merge? I'm hoping this will begin to unblock uwsgi.

**Comment by jakirkham** [comment]
Thoughts @conda-forge/jansson ? 🙂

**Comment by corydolphin** [comment]
Hey @mariusvniekerk @xhochy I'm sure you both are incredibly busy. This should be a quick PR to review and will allow more ARM64 native packages to be built for mac. In particular, this unblocks building/upgrading uwsgi. If either of you have a few minutes to take a look it would be much appreciated!

**Comment by jakirkham** [comment]
Let's give them some time. Think it is late in the day for both of them

**Comment by corydolphin** [comment]
> Let's give them some time. Think it is late in the day for both of them

Friendly bump. Does this change look reasonable and something you'd be open to merging?

File: recipe/conda_build_config.yaml
Comment (xhochy) [inline recipe/conda_build_config.yaml]: Why was this added?

**Review by xhochy — DISMISSED** [review]

**Comment by corydolphin** [comment]
@conda-forge-admin please rerender

File: recipe/conda_build_config.yaml
Comment (corydolphin) [reply recipe/conda_build_config.yaml]: Removed and rerendered. This was due to local testing differences. Apologies for the noise.

File: recipe/build.sh
Comment (jakirkham) [inline recipe/build.sh]: Seems an extra blank line crept in, suggesting we drop it

```suggestion
```

**Comment by corydolphin** [comment]
Looks like I don't have permissions to retry the build which appears to have flaked pulling from Github.
@conda-forge-admin please rerender

**Comment by jakirkham** [comment]
> Looks like I don't have permissions to retry the build which appears to have flaked pulling from Github.

Would take a look at [this bot request]( https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-restart-ci )

**Comment by jakirkham** [comment]
@conda-forge-admin, please restart CI

**Comment by corydolphin** [comment]
> > Looks like I don't have permissions to retry the build which appears to have flaked pulling from Github.
> 
> Would take a look at [this bot request](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-restart-ci)

Thanks!

**Comment by corydolphin** [comment]
Hey @xhochy Are you able to help take another look at this? I believe the feedback has been addressed. This should help unblock downstream dependencies from supporting ARM64 which would be great for the ecosystem.

Let me know if I can buy you a coffee/donate to thank you!

**Comment by jakirkham** [comment]
@xhochy, could you please give this another look (when you have a moment :)?

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render

**Review by xhochy — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks Uwe! 🙏

**Pull request merged**
